Continuous Delivery and Operation Excellence/Continuous Delivery.md

```diff
@@ -0,0 +1,54 @@
+# Content gathered for Continuous Delivery
+
+## Delivery formats
+* Webinar 
+* Self-service content 
+* Office Hours
+* Lab
+
+## Plans
+Plan | Topic | Delivery Format 
+---- | ----- | ---------------
+Week 1 | Fast and Continuous Delivery in SaaS SDLC | Webinar
+
+## Plans Details
+### DevOps culture | Webinar | Kickoff for this item
+    - DevOps Cultures
+    - Benefit of DevOps culture in related to SaaS Journey:
+        ○ Ref the Continuous Delivery section in ISV SaaS Playbook(Grow Your ISV Business with SaaS: Microsoft Practices Development Playbook)
+    - How DevOps and automation can improve productivities, security and fast go to market, A/B testing
+    - Sharing resources
+    - Microsoft tools & Open source tools sharing
+    - Gather next interests
+    - Asking for input on interests on Office Hours, frequency 
+    - Asking for input of common challenges in speeding up SDLC
+
+## Top of minds & Reference resources

```

Comment:
Might also be worth having some external resources like DORA State of DevOps. 

Could also add links to the MS Learn DevOps courses.

Reply:
Already there. But I think links is detached. Will fix